docs/status.rst

```diff
@@ -0,0 +1,38 @@
+..
+  SPDX-License-Identifier: CC-BY-4.0
+  Copyright Contributors to the OpenColorIO Project.
+
+Component Status
+================
+
++-------------------------------+----------------+----------------------------------------------------------------------------------+
+| Component                     | Status         | Notes                                                                            |
```

Comment:
This table seems to be duplicated in the Readme?

Reply:
Yes it was on purpose but I can drop it!